lib/DefaultResolver.php

```diff
@@ -214,6 +218,8 @@ private function doResolve($name, array $types, $options) {
             $this->config = (yield \Amp\resolve($this->loadResolvConf()));
         }
 
+        $config = $this->loadEnvConfig();
```

Comment:
Why shouldn't we? I think we can't access the original `RES_OPTIONS` anyway. Allowing it to change allows changing the defaults at runtime, e.g. applying options fetched from an external source.